<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BitstreamVersionSelector</class>
 <widget class="QDialog" name="BitstreamVersionSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>197</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select codec</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Codec</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="hm40">
            <property name="text">
             <string>HM 4.0</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="hm52">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>HM 5.2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="hm100">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>HM 10.0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="hm120">
            <property name="text">
             <string>HM 12.0 or later</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="av1">
            <property name="text">
             <string>AV1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BitstreamVersionSelector</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BitstreamVersionSelector</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
